src/main/java/ui/windows/stats/StatsDisplay.java

```diff
@@ -30,6 +40,26 @@ public StatsDisplay(StatsDelegator delegator) {
         initializeGUIComponents();
     }
 
+    /**
+     * Update this instance's labels to reflect updated values in data.
+     * @param data the data to update the labels with
+     */
+    @Override
+    public void updateView(StatsDisplayData data) {
+        if (isErrorData(data)) {
+            nameLabel.setText("Error encountered. Check console for more information.");
+            gamesWonLabel.setText(EMPTY_LABEL);
+            gamesLostLabel.setText(EMPTY_LABEL);
+            nextPlayer.setText(EMPTY_LABEL);
+        } 
+        else {
+            nameLabel.setText("User: " + data.getName());
+            gamesWonLabel.setText("Games Won: " + data.getGamesWon());
+            gamesLostLabel.setText("Games Lost: " + data.getGamesLost());
+            nextPlayer.setText("<html><br>Enter a new username to see statistics.</html>");
```

Comment:
Do you need the html tags if you have the WrappingLabel class?

Reply:
good catch, no it shouldnt need them